src/navigation/TabsNavigator.js

```diff
@@ -4,9 +4,9 @@ import { AsyncStorage, Text, StyleSheet, View } from "react-native";
 import { createBottomTabNavigator, createAppContainer } from "react-navigation";
 
 // Redadalertas
-import Events from "navigation/tabs/Events";
-import Report from "navigation/tabs/Report";
-import Settings from "navigation/tabs/Settings";
+import EventsTab from "navigation/tabs/Events";
+import ReportTab from "navigation/tabs/Report";
+import SettingsTab from "navigation/tabs/Settings";
```

Comment:
You didn't change the object created between lines 19 and 23, so I don't think this page will work correctly with this change.

Reply:
Ok I'll make this change, but I don't think it will do anything until I add Redux, so I'll put it in the next PR.